docs/tutorials/data-loading/local-files.md

```diff
@@ -19,13 +19,22 @@ You need the following software installed:
 * docker-compose
 * Node.js
 
+!!! Note
+    Shell commands should be run in a Bash shell.
+    If you are using Windows, we recommend using Git Bash.
+
 ## Setup
 
 To run the example code, clone the [qlik-oss/core-data-loading](https://github.com/qlik-oss/core-data-loading) Git
-repository. Check out the repo documentation to get familiar with the content and structure.
+repository. Before you continue, look at the documentation to get familiar with the content and structure of
+the repository.
+
+For this example, you only need a single instance of Qlik Associative Engine. All necessary data files are mounted when
+you start Qlik Associative Engine.
+
+To begin, you must accept the `ACCEPT_EULA`.
```

Comment:
Nit - Same here: One would accept the EULA not really the `ACCEPT_EULA` env var.